contrib/IECoreMantra/src/IECoreMantra/ProceduralPrimitive.cpp

```diff
@@ -179,7 +184,8 @@ void IECoreMantra::ProceduralPrimitive::addVisibleRenderable( VisibleRenderableP
 			addGeometry(gdp, 0.0f);
 			while ( !m_renderer->m_motionTimes.empty() )
 			{
-				setPreTransform( convert< UT_Matrix4T<float> >(m_renderer->m_motionTransforms.front()),
```

Comment:
Oh, we're missing that convertsion type. It should be like this one: https://github.com/ImageEngine/cortex/blob/master/include/IECoreHoudini/Convert.h#L109 but from an `M44f`. I don't mind if you leave it the way you have it, or if you want to add that extra convert template.